upup/pkg/fi/cloudup/vsphere/vsphere_cloud.go

```diff
@@ -36,6 +40,18 @@ func (c *VSphereCloud) ProviderID() fi.CloudProviderID {
 	return fi.CloudProviderVSphere
 }
 
+func NewVSphereCloud(spec *kops.ClusterSpec) (*VSphereCloud, error) {
+	server := spec.VSphereServer
+	datacenter := spec.VSphereDatacenter
+	cluster := spec.VSphereResourcePool
+	username := os.Getenv("VSPHERE_USERNAME")
+	password := os.Getenv("VSPHERE_PASSWORD")
+
```

Comment:
What will happen if the environment variable is not set? At which point will the user start to see the problem? 
Will it be better to check these env variables for emptiness in create_cluster.go?